src/Hng.Domain/Entities/Product.cs

```diff
@@ -2,8 +2,18 @@
 namespace Hng.Domain.Entities;
 public class Product : EntityBase
 {
+    public Product()
+    {
+        Id = Guid.NewGuid();
```

Comment:
This `Id = Guid.NewGuid();` is okay but I personally prefer the database generates it.

Reply:
Id initialization removed